jenkins/jobs/glusterfs-rpms.yml

```diff
@@ -0,0 +1,51 @@
+- job:
+    name: glusterfs-rpms-test
```

Comment:
Do we need to prefix job by "glusterfs", since that's kinda "obvious" givne we do not share jenkins with any others project ?